system/config/Config.cfc

```diff
@@ -839,6 +839,8 @@ component {
-amount-down,sort-amount-down-alt,sort-amount-up,sort-amount-up-alt,sort-down,sort-numeric-down,sort-numeric-down-alt,sort-numeric-up,sort-numeric-up-alt,sort-up,soundcloud,sourcetree,spa,space-shuttle,speakap,speaker-deck,spell-check,spider,spinner,splotch,spotify,spray-can,square,square-full,square-root-alt,squarespace,stack-exchange,stack-overflow,stackpath,stamp,star,star-and-crescent,star-half,star-half-alt,star-of-david,star-of-life,staylinked,steam,steam-square,steam-symbol,step-backward,step-forward,stethoscope,sticker-mule,sticky-note,stop,stop-circle,stopwatch,store,store-alt,strava,stream,street-view,strikethrough,stripe,stripe-s,stroopwafel,studiovinari,stumbleupon,stumbleupon-circle,subscript,subway,suitcase,suitcase-rolling,sun,superpowers,superscript,supple,surprise,suse,swatchbook,swimmer,swimming-pool,symfony,synagogue,sync,sync-alt,syringe,table,table-tennis,tablet,tablet-alt,tablets,tachometer-alt,tag,tags,tape,tasks,taxi,teamspeak,teeth,teeth-open,telegram,telegram-plane,temperature-high,temperature-low,tencent-weibo,tenge,terminal,text-height,text-width,th,th-large,th-list,the-red-yeti,theater-masks,themeco,themeisle,thermometer,thermometer-empty,thermometer-full,thermometer-half,thermometer-quarter,thermometer-three-quarters,think-peaks,thumbs-down,thumbs-up,thumbtack,ticket-alt,times,times-circle,tint,tint-slash,tired,toggle-off,toggle-on,toilet,toilet-paper,toolbox,tools,tooth,torah,torii-gate,tractor,trade-federation,trademark,traffic-light,train,tram,transgender,transgender-alt,trash,trash-alt,trash-restore,trash-restore-alt,tree,trello,tripadvisor,trophy,truck,truck-loading,truck-monster,truck-moving,truck-pickup,tshirt,tty,tumblr,tumblr-square,tv,twitch,twitter,twitter-square,typo3,uber,ubuntu,uikit,umbrella,umbrella-beach,underline,undo,undo-alt,uniregistry,universal-access,university,unlink,unlock,unlock-alt,untappd,upload,ups,usb,user,user-alt,user-alt-slash,user-astronaut,user-check,user-circle,user-clock,user-cog,user-edit,user-friends,user-graduate,user-injured,user-lock,user-md,user-minus,user-ninja,user-nurse,user-plus,user-secret,user-shield,user-slash,user-tag,user-tie,user-times,users,users-cog,usps,ussunnah,utensil-spoon,utensils,vaadin,vector-square,venus,venus-double,venus-mars,viacoin,viadeo,viadeo-square,vial,vials,viber,video,video-slash,vihara,vimeo,vimeo-square,vimeo-v,vine,vk,vnv,voicemail,volleyball-ball,volume-down,volume-mute,volume-off,volume-up,vote-yea,vr-cardboard,vuejs,walking,wallet,warehouse,water,wave-square,waze,weebly,weibo,weight,weight-hanging,weixin,whatsapp,whatsapp-square,wheelchair,whmcs,wifi,wikipedia-w,wind,window-close,window-maximize,window-minimize,window-restore,windows,wine-bottle,wine-glass,wine-glass-alt,wix,wizards-of-the-coast,wolf-pack-battalion,won-sign,wordpress,wordpress-simple,wpbeginner,wpexplorer,wpforms,wpressr,wrench,x-ray,xbox,xing,xing-square,y-combinator,yahoo,yammer,yandex,yandex-international,yarn,yelp,yen-sign,yin-yang,yoast,youtube,youtube-square,zhihu";
```

Comment:
I think this should be an array, no need to have as a string list and convert every time?